test/test_ops.py

```diff
@@ -125,13 +131,146 @@ def test_inplace_gradgrad(self, device, dtype, op):
         self._gradgrad_test_helper(device, dtype, op, self._get_safe_inplace(op.get_inplace()))
 
 
-class TestOut(TestCase):
+class TestCommon(JitCommonTestCase):
```

Comment:
This new parent class contains the functionality that test_jit.py and test_ops.py can share